### ISSUE TITLE
[Package]: xclip clipboard utility

### ISSUE THREAD

**Issue by snooppr** [termux/termux-packages#15385]
### Why do you want to add this package to Termux?

there are utilities that require xclip to work.

### Source code URL

https://github.com/astrand/xclip

### Packaging policy acknowledgement

- [X] The requested package complies with [Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy).

### Additional information

![xclip](https://user-images.githubusercontent.com/61022210/221752813-c908cf8c-597f-4075-b706-59c619a85c58.jpg)

**Comment by IntinteDAO** [comment]
You have this package in repository

If some apps required it by default, maybe we should add it to dependencies

**Comment by j0hnd3v** [comment]
https://github.com/termux/termux-packages/pull/7193

**Comment by snooppr** [comment]
> `pkg install x11-repo`
> 
> `pkg install xclip`

ok, i updated and installed x11-repo and xclip.  I tried to run xclip with any display variable, I get an error.  Is it possible to get xclip to work in Termux cli without vnc and such?

$ export DISPALY=localhost:0
or
$ export DISPALY=":1"
$ echo test | xclip -i
Error: Can't open display: :0

**Comment by snooppr** [comment]
> For completely headless setup Termux:API should be used instead.

Yes, I know about api termux and that it is convenient to manage the clipboard there.  But I came across a very popular 3rd party python CLI utility which is for GNU/Linux, works almost fine in Termux which uses xclip.  Therefore, it constantly gives an error and, in part, copy / paste does not work.

Thanks for the help anyway.
